waf trys to get package's configuration by native ncurses6-config.
it will make native header files and library be used.

Signed-off-by: Bian Naimeng <biannm@cn.fujitsu.com>

--- samba-4.1.12.orig/source3/wscript_configure_system_ncurses	2013-12-05 18:16:48.000000000 +0900
+++ samba-4.1.12/source3/wscript_configure_system_ncurses	2015-04-29 16:12:22.619000250 +0900
@@ -2,14 +2,6 @@ import Logs, Options, sys
 
 Logs.info("Looking for ncurses features")
 
-conf.find_program('ncurses5-config', var='NCURSES_CONFIG')
-if not conf.env.NCURSES_CONFIG:
-    conf.find_program('ncurses6-config', var='NCURSES_CONFIG')
-
-if conf.env.NCURSES_CONFIG:
-    conf.check_cfg(path=conf.env.NCURSES_CONFIG, args="--cflags --libs",
-               package="", uselib_store="NCURSES")
-
 conf.CHECK_HEADERS('ncurses.h menu.h panel.h form.h', lib='ncurses')
 
 conf.CHECK_FUNCS_IN('initscr', 'ncurses')
